services/cad-service/geometry.py

```
@@ -0,0 +1,121 @@
+"""Shared CAD utilities for FastAPI and gRPC surfaces."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import Enum
+from io import BytesIO
+from typing import Dict, Iterator
```

Ruff (0.14.1)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)